CalibPPS/ESProducers/plugins/PrintTotemDAQMapping.cc

```diff
@@ -30,33 +30,34 @@ class PrintTotemDAQMapping : public edm::one::EDAnalyzer<> {
 private:
   /// label of the CTPPS sub-system
   std::string subSystemName;
-  edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken_;
-  edm::ESGetToken<TotemAnalysisMask, TotemReadoutRcd> maskToken_;
+  edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken;
```

Comment:
I see you change from `mappingToken` to `mappingToken_` this is the opposite of what CMSSW naming rules suggest. See 2.9 in https://cms-sw.github.io/cms_coding_rules.html (and then technically the `subSystemName` should have been `subSystemName_`

Reply:
Besides the cosmetics, these could be `const`, no?